docs/src/developer-guide/nodejs-api.md

```diff
@@ -101,7 +101,7 @@ The `ESLint` constructor takes an `options` object. If you omit the `options` ob
 ##### Linting
 
 * `options.allowInlineConfig` (`boolean`)<br>
-  Default is `true`. If `false` is present, ESLint suppresses directive comments in source code. If this option is `false`, it overrides the `noInlineConfig` setting in your configurations.
+  Default is `true`. If `false` is present, ESLint suppresses directive comments in source code. If this option is `true`, it overrides the `noInlineConfig` setting in your configurations.
```

Comment:
I think this was correct. `allowInlineConfig=true` does not override `noInlineConfig=true`, while `allowInlineConfig=false` does override `noInlineConfig=false`.

Reply:
OK, so I close then, thank you!